interface/src/app/gene/network/network.js

```diff
@@ -390,9 +401,16 @@ angular.module('adage.gene.network', [
                 return g.id;
               });
 
-              ExpressionValue.get(
-                {'gene__in': geneID.join(), 'sample__in': sampleID.join(),
-                  'order_by': 'gene'},
+              ExpressionValue.post(
+                {},
+                // Angular does not serialize POST data the way we would expect
+                // so we have to do it manually here. For more details, see
+                // https://github.com/angular/angular.js/issues/6039#issuecomment-113502695
+                $httpParamSerializerJQLike({
```

Comment:
I found this link helpful in understanding `$httpParamSerializerJQLike` service in AngularJS:
https://docs.angularjs.org/api/ng/service/$httpParamSerializerJQLike